clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -47,7 +47,17 @@
 #include "utils.h"
```

Comment:
Re the PR description: I suppose you mean s/i6404/#6406/

Reply:
Yup. I've fixed the description.